lib/open_pgp/public_key_packet.ex

```diff
@@ -137,14 +137,22 @@ defmodule OpenPGP.PublicKeyPacket do
     {packet, next}
   end
 
-  # Support only RSA as of version 0.5.x
+  # Support only RSA and ElGamal as of version 0.6.x
```

Comment:
Maybe `@since` would be a useful annotation for some of this?

Reply:
This is the same function, but it can support new algo. AFAIK the `@since` module attribute is to annotate the function, not a specific function signature/arity. Correct me if I'm wrong.

> @since is an optional attribute that annotates which version the function was introduced.